docs/plugins/discovery-ec2.asciidoc

```diff
@@ -0,0 +1,215 @@
+[[discovery-ec2]]
+=== EC2 Discovery Plugin
+
+The EC2 discovery plugin uses the https://github.com/aws/aws-sdk-java[AWS API] for unicast discovery.
+
+[[discovery-ec2-install]]
+[float]
+==== Installation
+
+This plugin can be installed using the plugin manager:
+
+[source,sh]
+----------------------------------------------------------------
+sudo bin/plugin install discovery-ec2
+----------------------------------------------------------------
+
+The plugin must be installed on every node in the cluster, and each node must
+be restarted after installation.
+
+[[discovery-ec2-remove]]
+[float]
+==== Removal
+
+The plugin can be removed with the following command:
+
+[source,sh]
+----------------------------------------------------------------
+sudo bin/plugin remove discovery-ec2
+----------------------------------------------------------------
+
+The node must be stopped before removing the plugin.
```

Comment:
@clintongormley WDYT? ^^^ This has been added for each plugin.

I'm +1 for keeping this information here even if it's redundant.